cookbooks/prescribed_velocity_ascii_data/make_ascii_files_from_png.py

```diff
@@ -0,0 +1,65 @@
+from PIL import Image
```

Comment:
This is an awesome little script. I've often wished we had something like this for any number of purposes. Can you add a comment to the top what it does and how to call it?